phpBB/adm/style/acp_ext_list.html

```diff
@@ -77,6 +78,7 @@
 			<td style="text-align: center;">
 				<!-- IF disabled.S_VERSIONCHECK -->
 				<strong <!-- IF disabled.S_UP_TO_DATE -->style="color: #228822;"<!-- ELSE -->style="color: #BC2A4D;"<!-- ENDIF -->>{disabled.META_VERSION}</strong>
+				<!-- IF not disabled.S_UP_TO_DATE --><i class="fa fa-exclamation-circle" aria-hidden="true"></i><!-- ENDIF -->
```

Comment:
Maybe `icon-red`

Reply:
Yeah but add a unique class maybe "disabled-ext-icon" and color it in admin.css pls

Reply:
To be honest, I almost believe that it looks good as it is because the red looks just... a bit weird there, IMHO. What do you think?

![Screenshot](https://user-images.githubusercontent.com/1637630/30247854-c54770e6-961c-11e7-9d22-04e42477d62a.png)

Reply:
For me, the red is better attracting user attention.